mod_reforged/hooks/items/weapons/named/named_javelin.nut

```diff
@@ -5,4 +5,11 @@
 		this.m.BaseWeaponScript = "scripts/items/weapons/javelin";
 		create();
 	}
+
+	o.onEquip = function()
```

Comment:
As you are not changing the costs, the addition of this onEquip function should be removed from our hook.